Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
Oh, I figured we'd want FluentAssertions first, since it would make my PR valid. Otherwise, I have weak tests!